docs/sphinx/conf.py

```diff
@@ -0,0 +1,95 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+import os
+import sys
+# Ensure project root is on the path so `cloudos_cli` can be imported
+sys.path.insert(0, os.path.abspath('../..'))
+
+# -- Project information -----------------------------------------------------
+
+project = 'CloudOS CLI'
+copyright = '2025, Lifebit AI'
+author = 'Lifebit AI'
+
+# The full version, including alpha/beta/rc tags
+release = '1.0.0'
```

Comment:
```suggestion
release = '2.59.0'
```
Shouldn't this be the cloudos-cli version? Or it is the docs version. Also, is there a way to automatically pick it up? If it is the cloudos-cli version you can import the version value with:
```
from cloudos_cli._version import __version__
```